app/reports/budgetComparison/services/budgetComparisonReportCalculations.js

```diff
@@ -634,7 +632,7 @@ class BudgetComparisonReportCalculations {
 
         budgetScenarios.ids.forEach(function(budgetScenarioId) {
           var budgetScenario = budgetScenarios.list[budgetScenarioId];
-          budgetScenario.description = budgetScenario.isSnapshot ? `SNAPSHOT - ${dateToCalendar(budgetScenario.creationDate)} - ${budgetScenario.name}` : budgetScenario.name;
+          budgetScenario.description = budgetScenario.name;
```

Comment:
Will the DO lose anything without created date in the description?

Reply:
They shouldn't.

Previously, the snapshot name on create was just the scenario name "Live Data". Which is why I had to append the SNAPSHOT label and creation date.

Now, the snapshot name on create is "Budget Request - MM/dd/YYYY"